src/sentry/api/endpoints/group_event_details.py

```diff
@@ -156,6 +156,41 @@ def get(self, request: Request, group: Group, event_id: str) -> Response:
                         event = group.get_helpful_event_for_environments(environments)
             else:
                 return Response(status=404)
+        elif event_id == "recommended":
+            if features.has(
```

Comment:
instead of repeating this code, couldn't we just do `elif event_id in ("helpful", "recommended")` above?

Reply:
I added it to have more complete test coverage before we remove the 'helpful' endpoint